hazelcast/src/main/java/com/hazelcast/jet/impl/execution/init/Contexts.java

```diff
@@ -378,5 +382,51 @@ public int localProcessorIndex() {
         public int globalProcessorIndex() {
             return globalProcessorIndex;
         }
+
+        public MetricsContext metricsContext() {
+            return metricsContext;
+        }
+    }
+
+    public static Container container() {
+        return CONTEXT.get();
+    }
+
+    @Nonnull
+    public static Processor.Context getThreadContext() {
```

Comment:
nit: I would rather call it processor context than thread context.

Reply:
I'd rather keep the current name. I think the fact that it's a context tied to current thread is more important than the fact that it's a processor context. The latter is also known from the return type.